var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -104,6 +104,9 @@ class PyNumpy(PythonPackage):
     # https://github.com/numpy/numpy/pull/13132
     patch('blas-lapack-order.patch', when='@1.15:1.16')
 
+    # Add Fujitsu Fortran compiler
+    patch('add_fj_compiler.patch', when='%fj')
```

Comment:
You should be careful about what versions you apply this patch to. Does this work for all versions of `py-numpy`? It definitely doesn't work for all versions of Python. You're using f-strings, which were added to Python 3.6 and aren't present in older versions of Python. Numpy 1.19+ is Python 3.6+ only, so that's fine, but older versions of Numpy support older versions of Python.

Reply:
Thanks for comment. I created patches to apply to each version.